plugins/katello/developers.md

```diff
@@ -45,53 +42,56 @@ There are a few important states that issues can be in that are worth knowing ab
 
 #### General Flow for New or Needs More Information
 
-The general flow is to open an issue and the runner to exam the issue, typically reading the general idea aloud. This allows for any developers with knowledge to chime in with any relevant info. The runner attempts to answers the following questions:
+The general flow is to open an issue and the runner to examine the issue, typically reading the issue number and general idea aloud. This allows for any developers with knowledge to chime in with any relevant info. The runner attempts to answer the following questions:
 
     * Does this issue need more information from the user?
-        * If yes, leave a comment asking for that information
+        * If yes, leave a comment asking for that information, set the 'Needs More Information' status
     * Does this issue need a developer to investigate it further to properly triage it?
         * If yes, set status to 'Needs Design'
-    * What release should the issue be targeted for? (e.g. next z-stream, current release, next release, backlog)
+    * Which release should the issue be targeted for? (e.g. next z-stream, current release, next release, or no release (implicit backlog))
     * Does any developer want to own it?
         * If yes, assign it
+    * Does the issue have downstream information?
+        * If yes, edit the description to remove the downstream bits
     * Issue category?
+    * Issue priority?
+    * Issue difficulty?
 
 #### General Flow for Ready for Testing
 
   * Will this PR get completed by the next release?
-  * What release should the issue be targeted for? (e.g. next z-stream, current release, next release, backlog)
+  * What release should the issue be targeted for? (e.g. next z-stream, current release, next release)
 
 #### General Flow for Closed
 
-  * Should this issue be targeted at the next z-stream? Current RC phase? Or just throw it in the "next" release (aka develop or master)
+  * What release should the issue be targeted for? (e.g. next z-stream, current release, next release)
 
 ### Notes
 
 A few notes about some automatic operations that are performed by the prprocessor that help facilitate this workflow.
 
-  * All issues are set to an empty release, which again represents "untriaged"
-  * Any issues on the backlog that a PR is opened for has its release set to 'empty' to ensure that it flows through the triage process
-  * Developers that believe an issue should be considered for a release, should set the release to 'empty' and show up to triage to argue their case
+  * All issues are set to an empty release and untriaged
+  * Any issues on the backlog that a PR is opened for has its release set to 'empty' and triaged to false to ensure that it flows through the triage process
+  * Developers that believe an issue should be considered for a release should set triaged to false and show up to triage to argue their case
```

Comment:
So, this goes back to the old way of doing triage that we had issues with (and hence the current process). What does the following mean now:

 * On a release, triage unset
 * Not on a release, triage set

The original problem we were solving with the current method (and simplifying) was that we did not need a triaged flag. Rather, release means triaged. I believe the only flaw we ever saw was that users could directly set issues on a release which did happen but rarer. I think given the two scenarios above, if we are going to use the triaged flag, the process needs to be modified to account for those states.

Reply:
> On a release, triage unset

This basically means a user set the target release (on accident or on purpose), almost like, "please put it on this release". When we see it in triage, we'll set the triage flag and update the target release (or leave it).

> Not on a release, triage set
If the issue is open, it's implicitly on the backlog. If it's closed, it's implicitly in the recycle bin. When issues move to ready for testing, the triage flag should be cleared by prprocessor.

Reply:
I felt like the backlog was a nice and specific abstraction for putting things in that bucket that was obvious for users. It would read in the issue "Backlog" for example. Now I feel things are less obvious to users and to developers looking through issues.

I am still trying to wrap my head around what the "Triaged" flag now means. Triaged used to mean the team has reviewed and assigned an issue to a release. If a user wanted it re-considered they could unset the release or pop it off the backlog. 

The doc here says "Developers that believe" but what about users? :)

Reply:
Hah, so @jlsherrill is the real culprit here? Let's see if pinging him a few times gets him to pop in here.

Reply:
I would just say "leave Triaged unchecked" instead of "set triage to false" as it is a passive action